ChangeLog

```diff
@@ -1,3 +1,45 @@
+0.33
```

Comment:
we should drop ChangeLog.
git log 0.32..0.33 .

Reply:
You mean: you see no need in keeping a `ChangeLog` file as there's `git log` already, and the ChangeLog adds almost nothing to what one could get from `git log [--format=...]`?

Reply:
right.

Reply:
Not going to comment on cloud-utils, but when it comes to cloud-init, I strongly disagree. A changelog is for a consumer of the project, not a developer of the project. They're not likely to have it already cloned from a git repo, and that information isn't otherwise contained in any of the release artifacts. E.g., for me, if I noticed a regression in a dependency, the first thing I do (after looking for a pre-existing bug) is to google the changelog to see what changed. I don't want to have to git clone something just to see whats new.

Reply:
@smoser if you are OK with cutting a release, could you approve this PR? I assume your comment here is non-blocking, but others may be on the fence on this PR, waiting for us to resolve the conversation.

I think we can discuss the ChangeLog or the release process in general, but I'd prefer to do it after the release. We can open a bug for tracking.

Thanks!

Reply:
thats fine to clean up ChangeLog later. I thought I had deleted it from cloud-init at some point in the past.  I know I did that somewhere in some package....